open_samus_returns_rando/files/levels/s100_area10.lua

```diff
@@ -271,6 +271,9 @@ function s100_area10.OnSubAreaChange(_ARG_0_, _ARG_1_, _ARG_2_, _ARG_3_, _ARG_4_
     Game.SetSceneGroupEnabledByName("sg_egg02", true)
     Game.EnableEntity("LE_Baby_Hatchling")
   end
+  if _ARG_2_ == "collision_camera_007" then
+    Blackboard.SetProp("s100_area10", "s100_area10_discovered", "b", true)
+  end
```

Comment:
Just came to my mind: This is for compatibilty iff we find out how to have the whole map visible but unvisited? Because then the vanilla thing would be true already.

Reply:
I thought we weren't going to go that route if we added an option to have majors be visible on the map?

Reply:
If this is the vanilla property, why do you have to explicitly set it here?
E.g. start your game with a init where you do `Blackboard.SetProp("s100_area10", "s100_area10_discovered", "b", true)` this makes the Area 8 visible on the minimap selection from the beginning. Thus it should be already handled by the game itself as soon as you visit area 8.

Reply:
I don't know if this is a vanilla property or not. I set this so the elevator only becomes active if the elevator in area 8 has been used (ie entering the collision camera). Maybe it's possible to check for the elevator being used directly.

Reply:
That's pretty funny because you hit exactly the vanilla naming scheme. This property should be set by the game to true as soon as you have area 8 visible on your minimap. That's not what you actually wanted because now it should be sufficient to just dip into area 8 and you don't have to reach the elevator.